example/object_examples.py

```diff
@@ -0,0 +1,401 @@
+
+    if text != None and txyz != None:
+        tx, ty, tz = txyz
+        t = Text(text=text, font_family="arial", bounding_box=False, xyz=[-tx, -ty, 15], rotation=0, scale=0.007).write()
+
+    topVertices = [v + height if i % 3 == 2 else v for i, v in enumerate(btmShape)]
+    allVertices = btmShape + topVertices
+
+    bottom_faces = []
+    i = 0
+    for x in range(len(btmShape)-1):
+        if x % 3 == 2:
+            bottom_faces.append(i)
+            i += 1
+    p = bottom_faces.insert(0, len(bottom_faces))
+
+    top_faces = []
+    for index, i in enumerate(bottom_faces[1:]):
+        if index == 0:
+            top_faces.append(bottom_faces[0])
+        top_faces.append(bottom_faces[0]+1 + bottom_faces[i+1])
+
+    tempList = []
+    side_faces = []
+
+    for i in range(len(bottom_faces)-1):
+        between1 = bottom_faces[i+1]+1
+        list1 = [4, bottom_faces[i+1], between1]
+
+        mXtop_faces = top_faces[-1:][0]
+        between2 = top_faces[i+1]+1
+        if between2 > mXtop_faces:
+            between2 = top_faces[1]
+        list2 = [between2, top_faces[i+1]]
+
+        tempList = list1 + list2
+        side_faces.append(tempList)
+
+    allFaces = bottom_faces + top_faces + flatten(side_faces)
+
+    meshPlatform = Mesh(
+        vertices=allVertices,
+        faces=allFaces,
+        units="mm"
+    )
+    if text != None:
+        return meshPlatform, t
+    else:
+        return meshPlatform
+
+
+MeshExample0 = Platform(5, [481,160,13], btmShape="Example0")
+objList.append(MeshExample0)
+
+MeshExample1 = Platform(5, [475,10,13], btmShape="Example1")
+objList.append(MeshExample1)
+
+MeshBase0 = Platform(10, [0,0,0], btmShape="OveralShape")
+objList.append(MeshBase0)
+
+MeshBase1 = Platform(2, [0,0,10], btmShape="btmShape", text="LINE", txyz=[0+20,0+50,10])
+objList.append(MeshBase1)
+
+MeshBase2 = Platform(2, [150,0,10], btmShape="btmShape", text="ARC", txyz=[150+18,0+50,10])
+objList.append(MeshBase2)
+
+MeshBase3 = Platform(2, [150,150,10], btmShape="btmShape", text="POLYLINE", txyz=[150+0,150+50,10])
+objList.append(MeshBase3)
+
+MeshBase4 = Platform(2, [0,150,10], btmShape="btmShape", text="POINT", txyz=[0+15,150+50,10])
+objList.append(MeshBase4)
+
+MeshBase5 = Platform(2, [300,0,10], btmShape="btmShape", text="CIRCLE", txyz=[300+10,0+50,10])
+objList.append(MeshBase5)
+
+MeshBase6 = Platform(2, [300,150,10], btmShape="btmShape", text="POLYCURVE", txyz=[290,150+50,10])
+objList.append(MeshBase6)
+
+MeshBase7 = Platform(2, [450,0,10], btmShape="btmShape", text="MESH2", txyz=[450+10,0+50,10])
+objList.append(MeshBase7)
+
+MeshBase8 = Platform(2, [450,150,10], btmShape="btmShape", text="MESH1", txyz=[450+10,150+50,10])
+objList.append(MeshBase8)
+#Platform - end
+
+
+send_to_speckle(host="https://speckle.xyz", stream_id="d2e38baf76", objects=objList, msg="Objects")
```

Comment:
hey @UserDevtec 

please remove the references to speckle.xyz and the specific `stream_id`

Otherwise i'd be happy to take this.